### PR TITLE
add community & governance pages

### PR DESCRIPTION
**Pull request by TasoOneAsia**
## Proposed Changes

This pull requests renames `en/support.html` -> `en/community.html` and adds content around community and governance for open-source NGINX projects. 

Also removes the support section per the governing-committee 

### New Files

* Governance - (en/docs/governance.xml)

### CODE REVIEW

**Comment by lcrilly** [comment]
Adding two new menu items seems excessive. I suggest reworking the existing support page as a gateway to these topics.

**Comment by lcrilly** [comment]
Please also note the commit message style guide https://nginx.org/en/docs/contributing_changes.html#formatting_changes

**Comment by pluknet** [comment]
Thanks for your work, please explain why it is needed.
I don't think though that nginx.org is the best place to place various marketing, commercial or other non-technical stuff, please avoid doing this.  That kind of stuff is probably better hosted on nginx.com or elsewhere.  Also, removing high phrasing should allow to understand in a more clear way, what this kind of content is actually meant for.  I believe, text can be trimmed down several times such way, leaving only links and some short intro.

To clarify, nginx.org is used to host reference documentation in dry technical language.  I'd expect something similar from this change.

**Comment by TasoOneAsia** [comment]
@lcrilly, I adjusted the commit / commit desc to match your feedback.  I wasn't sure if you'd prefer that the new pages' indentation match the current precedent; if so, I can amend the commit again doing so. 

@pluknet, these changes were authored by the broader NGINX community team as a way of reaffirming NGINX commitments to the broader OSS community and governance of NGINX open-source projects. Outside of the mailing lists, this is one of the sole ways of communicating effectively with that community for the time being.

**Review by pluknet — CHANGES_REQUESTED** [review]
Please following existing style, in no particular order:
- missing copyright notice
- no leading spaces, except in special tags like <programlisting> and <example>
- crop lines at 80 characters
- maintain empty line balance
- avoid mixing with unrelated and/or breaking changes
Currently all points are violated.
For a guideline, see other articles such as xml/en/docs/beginners_guide.xml

File: xml/en/docs/community.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<!--
+  Copyright (C) F5, Inc.
+  -->
+
+<!DOCTYPE article SYSTEM "../../../dtd/article.dtd">
+
+<article name="Welcome to the NGINX Community!"
+         link="/en/docs/community.html"
+         lang="en"
+         rev="1">
```
Comment (y82) [inline xml/en/docs/community.xml:10]: 2 empty lines missing (<section> elements are separated by 2 emtpy lines)

File: xml/en/docs/community.xml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+<section>
+
+<para>
+Open source powers much of the technology world. NGINX has been a strong part
```
Comment (y82) [inline xml/en/docs/community.xml:14]: 2 sentences in one line (Each new sentence starts with a new line), needs to be fixed globally

File: xml/en/docs/governance.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
+<!--
+  Copyright (C) F5, Inc.
+  -->
+
+<!DOCTYPE article SYSTEM "../../../dtd/article.dtd">
+
+<article name="NGINX Open Source Governance"
+link="/en/docs/governance.html"
```
Comment (y82) [inline xml/en/docs/governance.xml:8]: wrong ident (should be aligned with 'name'), also below

File: xml/en/docs/governance.xml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+lang="en"
+rev="1">
+<section>
```
Comment (y82) [inline xml/en/docs/governance.xml:11]: 2 empty lines before section

File: xml/en/docs/governance.xml [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+<para>
+Open source changed the world.
+</para>
+
+<para>
+NGINX is proud to be part of that change, from our very beginning.
+</para>
+
+<para>
+NGINX is committed to continued involvement in open source. To achieve this
```
Comment (y82) [inline xml/en/docs/governance.xml:22]: each sentence starts with a new line, needs to be fixed globally

File: xml/en/docs/governance.xml [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+<list type="bullet">
+
+<listitem>
+We will be open, consistent, transparent, and fair in our acceptance of
+contributions.
+</listitem>
+
+<listitem>
+We will continue to enhance existing active projects
```
Comment (y82) [inline xml/en/docs/governance.xml:39]: either add full stop in the end of the sentence  or remove full stop in similar sentences above/below

File: xml/en/docs/governance.xml [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+</list>
+
+</section>
+
+<section name="NGINX Open Source Principles">
+
+<list type="bullet">
+
+<listitem>
+All contributions are reviewed by the technical committee for that project
```
Comment (y82) [inline xml/en/docs/governance.xml:67]: punctuation in the end of the sentence should be consistent with the previous list

**Review by y82 — CHANGES_REQUESTED** [review]

**Comment by y82** [comment]
Hi @TasoOneAsia, thank you for the patch, I left some style-related review notes.

As per content itself, I would appreciate if @lcrilly and @dwmcallister could give feedback or review and approve.

**Comment by TasoOneAsia** [comment]
Appreciate the review @y82

I've gone ahead and applied the styling feedback.

**Comment by TasoOneAsia** [comment]
Let me know if you would prefer I just amend the original commit

**Comment by y82** [comment]
Hi @TasoOneAsia , thank you for the new version, I went through it again and there is still not implemented feedback - I left comments again.

Yep, amending the original commit (i.e. keeping one commit) would be best at least for now as we've just started the gitub workflow.

**Comment by TasoOneAsia** [comment]
Amended commit as requested. 

Which styling items were you referring to specifically? Feel free to reopen the resolved conversation from the previous review if you spot something that still isn't resolved properly.

**Comment by dwmcallister** [comment]
I've looked over the content and approve this.

File: xml/en/docs/community.xml [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+We believe that healthy communities are core to innovation and open source
+sustainability.
+It is our mission to enable the NGINX community to engage with
+and develop solutions based on open and available NGINX technologies.
+Together we can power the future.
+</para>
+
+<para>
+Our community is your entry into an open world of information, collaboration,
+and discussion.  Thanks for joining us.
```
[the remark this answers is not in the document]
Comment (TasoOneAsia) [reply xml/en/docs/community.xml:42]: Good catch, for some reason my editor's formatting rules on print width decided to ignore just this line 🤷

File: xml/en/docs/community.xml [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+<para>
+Open source powers much of the technology world.
+NGINX has been a strong part of this movement from its inception.
+As the internet has grown, so have we, from our initial webserver to the
+myriad of projects involving JavaScript, Kubernetes, API gateways,
+OpenTelemetry, and WebAssembly.
+NGINX is committed to open source both in ethos and practicality, aiming to
+deliver reliable and cutting-edge technology within a healthy open source
+culture.
+Whether NGINX is part of your open source past or a step in your present, we welcome you into the
```
[the remark this answers is not in the document]
Comment (TasoOneAsia) [reply xml/en/docs/community.xml:24]: Good catch, for some reason my editor's formatting rules on print width decided to ignore just this line 🤷

File: xml/en/docs/governance.xml [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+</listitem>
+
+<listitem>
+We will not impose limits on the use of our projects.
+</listitem>
+
+</list>
+
+</section>
+
```
[the remark this answers is not in the document]
Comment (TasoOneAsia) [reply xml/en/docs/governance.xml:63]: @y82 Should I also adjust the new section in `support.xml` to be separated by two newlines? The existing section tags are currently separated by one newline.

File: xml/en/docs/community.xml [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+<listitem>
+<link url="https://twitter.com/nginxorg">Twitter</link>
+</listitem>
+
+<listitem>
+<link url="https://www.youtube.com/nginxinc">YouTube</link>
+</listitem>
+
+<listitem>
+<link url="/en/docs/governance.html">Governance</link>
```
Comment (lcrilly) [inline xml/en/docs/community.xml:84]: This link isn't somewhere you find the NGINX community. There seems no reason for it to be here, especially when it is prominently linked-to from the support page (see general comment about that).

I suggest replacing this item with a link to the NGINX community Slack.

File: xml/en/docs/community.xml [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+</listitem>
```
Comment (lcrilly) [inline xml/en/docs/community.xml:85]: Please also link to the mailing lists section of the support page.

File: xml/en/docs/community.xml [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+
+</list>
+
+</para>
+
+</section>
+
+
+<section name="Security">
```
Comment (lcrilly) [inline xml/en/docs/community.xml:94]: This section has no place here because there is already a 'security' link on the main nav menu. Please remove.

File: xml/en/docs/community.xml [hunk 14]
```diff
@@ -0,0 +1,136 @@
...
+<para>
+We advise users to run or update to the most recent release of this project.
+Older versions of this project may not have all enhancements and/or bug fixes
+applied to them.
+</para>
+
+</section>
+
+
+<section name="Reporting a Vulnerability">
```
Comment (lcrilly) [inline xml/en/docs/community.xml:105]: This information contradicts the information in the 'security' page. Please remove it. If you wish to change the vulnerability reporting guidelines then do so as a separate PR.

File: xml/en/docs/community.xml [hunk 15]
```diff
@@ -0,0 +1,136 @@
+<!--
+  Copyright (C) F5, Inc.
+  -->
+
+<!DOCTYPE article SYSTEM "../../../dtd/article.dtd">
+
+<article name="Welcome to the NGINX Community!"
```
Comment (lcrilly) [inline xml/en/docs/community.xml:7]: The information in this new page heavily overlaps with the content and intent of the 'support' page. There seems to be no good reason to introduce a new page here. I recommend that 'support' is renamed to 'community' (and on the menu) and the relevant sections of this file is carefully merged with the existing content.

A short-term redirect may be needed for /en/support.html

File: xml/en/docs/governance.xml [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+<para>
+Open source changed the world.
+</para>
+
+<para>
+NGINX is proud to be part of that change, from our very beginning.
+</para>
+
+<para>
```
Comment (lcrilly) [inline xml/en/docs/governance.xml:23]: This para reads:

> NGINX is committed to […]. To achieve this _goal_ we are making […] commitments …

So is "continued involvement in open source" a goal or a commitment? Having two levels of commitments is strange.

I suggest simplifying this to cover all of the remaining content on this page (commitments and principles).

> Our continued involvement in the open source community is encapsulated by our Core Commitments and Principles. These will help the longevity, transparency, and impact of our open source projects.

**Review by lcrilly — CHANGES_REQUESTED** [review]
Overall, this PR seems a little heavy handed, and I recommend a more elegant integration with existing content.

File: xml/en/docs/community.xml (hunk 15 above)
Comment (TasoOneAsia) [reply xml/en/docs/community.xml]: Mentioned explicitly in commit message as requested

**Comment by y82** [comment]
Hi @TasoOneAsia and @lcrilly,

I'm sorry I should have replied before the new version as now it might involve some extra work/rollout we possibly could avoid.

I'd still keep the 'support' page and add 'community' and 'governance' pages as new ones:

- .org is still a place for commercial docs (directives/variables & modules included in Plus are marked as "commercial subscription), so having a section about commercial support is correct, but placing it under 'community' is not

- 'Mailing lists' and 'Development' sections on the 'support' page are certainly about community,  but perhaps for now we'd leave them under 'support' which is also correct as what the mailing lists are about. At the same time, there's a requirement to add 'community' and 'governance' with some preapproved text, so I guess we can work on 'support' and 'community' content merge/modification later as step 2.

So, what to do with all this feedback :) I would suggest the following plan:

- revert support.xml, add links to 'community' and 'governance' as suggested by @lcrilly: https://github.com/nginx/nginx.org/commit/d17156aa6a871d7aaac6321a8fc8a621ac7167b1#diff-da0689a648f18d2e042a7fb777496df432fb54b5e60ef3f51f67cc23898594c2

- add 'community' as a separate page

- In the "Community Spaces" section of community.xml, add link to 1 - community Slack, 2 - to subsection "Mailing Lists" from support.xml, 3 - to subsection "Development" from support.xml

Hope this helps, and will be happy to collaborate to be more clear and to finally work out a good solution. Sorry again for my late feedback.

**Comment by annania** [comment]
> Hi @TasoOneAsia and @lcrilly,
> 
> I'm sorry I should have replied before the new version as now it might involve some extra work/rollout we possibly could avoid.
> 
> I'd still keep the 'support' page and add 'community' and 'governance' pages as new ones:
> 
> * .org is still a place for commercial docs (directives/variables & modules included in Plus are marked as "commercial subscription), so having a section about commercial support is correct, but placing it under 'community' is not
> * 'Mailing lists' and 'Development' sections on the 'support' page are certainly about community,  but perhaps for now we'd leave them under 'support' which is also correct as what the mailing lists are about. At the same time, there's a requirement to add 'community' and 'governance' with some preapproved text, so I guess we can work on 'support' and 'community' content merge/modification later as step 2.
> 
> So, what to do with all this feedback :) I would suggest the following plan:
> 
> * revert support.xml, add links to 'community' and 'governance' as suggested by @lcrilly: [d17156a#diff-da0689a648f18d2e042a7fb777496df432fb54b5e60ef3f51f67cc23898594c2](https://github.com/nginx/nginx.org/commit/d17156aa6a871d7aaac6321a8fc8a621ac7167b1#diff-da0689a648f18d2e042a7fb777496df432fb54b5e60ef3f51f67cc23898594c2)
> * add 'community' as a separate page
> * In the "Community Spaces" section of community.xml, add link to 1 - community Slack, 2 - to subsection "Mailing Lists" from support.xml, 3 - to subsection "Development" from support.xml
> 
> Hope this helps, and will be happy to collaborate to be more clear and to finally work out a good solution. Sorry again for my late feedback.

@lcrilly thoughts on this?

**Comment by TasoOneAsia** [comment]
Latest commit applies the governing committees requested changes

**Review by lcrilly — APPROVED** [review]

File: xml/en/community.xml [hunk 17]
```diff
@@ -5,18 +5,89 @@
...
+NGINX is committed to open source both in ethos and practicality, aiming to
+deliver reliable and cutting-edge technology within a healthy open source
+culture.
+Whether NGINX is part of your open source past or a step in your present,
+we welcome you into the future together.
+</para>
+
+<para>
+To learn more about our commitment, please visit
+the <link url="/en/docs/governance.html">Governance</link>
```
Comment (y82) [inline xml/en/community.xml:30]: bad link, should be:
``<link doc="docs/governance.xml">Governance</link>``

**Review by y82 — CHANGES_REQUESTED** [review]

File: xml/en/community.xml (hunk 17 above)
Comment (TasoOneAsia) [reply xml/en/community.xml:30]: Gotcha, so link directly to the underlying XML docs when appropriate.

File: xml/en/community.xml [hunk 18]
```diff
@@ -5,18 +5,89 @@
 
 <!DOCTYPE article SYSTEM "../../dtd/article.dtd">
 
-<article name="nginx: support"
-         link="/en/support.html"
+<article name="nginx: community"
+         link="/en/community.html"
          lang="en"
-         rev="10">
+         rev="11">
```
Comment (y82) [inline xml/en/community.xml:11]: I'd put version 1 instead of 11 (it's a new page and we might want to revive support.xml with better content at some time)

**Review by y82 — CHANGES_REQUESTED** [review]
1 - The style of the current commit msg is different - smth like that would be ok: "Added Community and Governance pages."
2- Version for community page should be 1 
After that we're good to go.

And although I understand that it's agreed to rename support.xml to community.xml and enhance support.xml later, I still don't appreciate it because: 1-we are breaking existing hyperlinks that lead to this page 2-Commercial support info will be missing. No actions -just leaving my thought here.

**Comment by y82** [comment]
yep, the redirect support > community will be configured as well

**Review by y82 — APPROVED** [review]
